corehq/messaging/smsbackends/connectid/views.py

```diff
@@ -27,3 +33,55 @@ def receive_message(request, *args, **kwargs):
     )
     process_incoming(msg, phone_obj)
     return HttpResponse("")
+
+
+@csrf_exempt
+@connectid_token_auth
+def connectid_messaging_key(request, domain):
```

Comment:
since this is a global URL, it won't have a domain argument, and neither will the `request` object

Reply:
Fixed. [3cb58a1](https://github.com/dimagi/commcare-hq/pull/35273/commits/3cb58a13326649332d7ed85619397b072f55ae2c)